src/Components/Quizbox/Quizbox.module.css

```diff
@@ -0,0 +1,34 @@
+.quizbox_container{
+    display: flex;
+    flex-direction: column;
+    margin-top: 35px;
+    border: 2px solid black;
+    width: 300px;
+    height: 300px;
+    position: relative;
+
+}
+
+.quizbox_information{
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+
+}
+
+/*.quizbox_information input{*/
```

Comment:
Als je dit weer aan gaat zetten en je wil dat dit item altijd onderin het quiz-box component komt te staan, kun je dit doen door de binnenste container een `display flex` te geven in direction `column`, en dan het element met alle inputs erin een `align-self: flex-end`